src/use_event_listener/index.jsx

```diff
@@ -1,18 +1,26 @@
-import { useEffect } from 'react';
+import { useEffect, useRef } from 'react';
 
 export const useEventListener = (
   ref,
   type,
-  listener,
-  options = { passive: true },
+  handler,
+  { passive, capture, once } = { passive: true },
 ) => {
+  const node = ref.current;
+  const handlerRef = useRef(null);
+
+  handlerRef.current = handler;
+
   useEffect(() => {
-    if (!ref.current) return;
+    if (!node) return;
```

Comment:
i think you need to create node variable inside useEffect, otherwise it's out of sync